src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
+  box-sizing: border-box;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition:
+      background-color 500ms,
+      transform 500ms;
+
+    &::before {
+      font-family: Arial, sans-serif;
+      font-size: 30px;
+      content: '';
```

Comment:
The `content` property in the `&::before` pseudo-element is set to an empty string. This might be unnecessary since it is later overridden in the `@for` loop with actual content. Consider removing this line if it's not needed.